content/en/docs/armory-admin/dinghy-enable.md

```diff
@@ -58,10 +58,78 @@ hal armory dinghy enable
 {{< /tabs >}}
 
 
+## Configuring SQL
+
+{{< include "early-access-feature.html" >}}
+
+The Dinghy service can use Redis to store relationships between pipeline templates and pipeline dinghy files. An external MySQL instance is highly recommended for production use because it can provide more durability for Pipelines as Code. If MySQL becomes unavailable, dinghy files will need to be updated in order to repopulate MySQL with the relationships.
```

Comment:
The Dinghy service can use Redis - I think we should change it to SQL?

Reply:
```
The Dinghy service can use MySQL to store relationships between pipeline templates and pipeline dinghy files. An external MySQL instance is highly recommended for production use because it can provide more durability for Pipelines as Code. If MySQL becomes unavailable, dinghy files will need to be updated in order to repopulate MySQL with the relationships.